src/track/track.h

```diff
@@ -410,6 +412,9 @@ class Track : public QObject {
     void coverArtUpdated();
     void beatsUpdated();
     void replayGainUpdated(mixxx::ReplayGain replayGain);
+    // This signal indicates that ReplayGain is being adjusted, and pregains should be
+    // adjusted in the opposite direction to compensate (no audible change).
+    void updateAndAdjustReplayGain(const mixxx::ReplayGain&);
```

Comment:
The naming is confusing and inconsistent with that of the remaining signals. How about `replayGainAdjusted`?

Reply:
done, and normalized the naming to always refer to "pregain"